src/math/matrix.c

```diff
@@ -0,0 +1 @@
+#include <math/matrix.h>
```

Comment:
There is nothing in this file as so far the matrix lib is header only. That might change later though, so I left this in.